dali/test/python/operator_2/test_warp_perspective.py

```diff
@@ -0,0 +1,372 @@
+# Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
We do a lot of parameter handling with tons of bug-prone ifs. I think we need to have some negative tests to check if we handle invalid argument combinations properly.

Reply:
I added a few tests that check throwing errors.